src/app/components/Hero/Hero.jsx

```diff
@@ -90,16 +87,16 @@ class Hero extends React.Component {
     const tags = utils.metaTagUnion(homepageTags);
 
     return (
-      <div className={this.props.className} style={bannerStyle}>
+      <div className="Hero" style={bannerStyle}>
         <DocMeta tags={tags} />
-        <div className={`${this.props.className}__container`}>
-          <div className={`${this.props.className}__text`}>
-            <div className={`${this.props.className}__text__HeroTitle`}>
+        <div className="Hero__container">
+          <div className="Hero__text">
+            <div className="Hero__text__HeroTitle">
               <h3>{heroData.title}</h3>
-              <p className={`${this.props.className}__text__HeroTitle__des`}>
+              <p className="Hero__text__HeroTitle__des">
                 {heroData.description}
               </p>
-              <p className={`${this.props.className}__text__HeroTitle__intro`}>
+              <p className="Hero__text__HeroTitle__intro">
```

Comment:
I would propose we pay a re-visit to this later on and establish a class naming convention similar to the toolkit.

Reply:
Definitely agree.